samples/graph-change-notification-team-channel/csharp/ChangeNotification/Model/TeamResource.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace ChangeNotification.Model
+{
+    using Newtonsoft.Json;
+    using System;
+
+    public class TeamResource
+    {
+        [JsonProperty(PropertyName = "@odata.type")]
+        public string ODataType { get; set; }
+
+        /// <summary>
+        /// SubscriptionId of resource SubscriptionId.
+        /// </summary>
+        [JsonProperty(PropertyName = "subscriptionId")]
+        public string SubscriptionId { get; set; }
+
+        /// <summary>
+        /// The type of resource ChangeType.
+        /// </summary>
+        [JsonProperty(PropertyName = "changeType")]
+        public string ChangeType { get; set; }
+
+         /// <summary>
+        /// The type of resource Team DisplayName.
+        /// </summary>
+        [JsonProperty(PropertyName = "displayName")]
+        public string DisplayName { get; set; }
+
+        public  DateTime? CreatedDate { get; set; }
```

Comment:
Summary and Json attribute missing for it.

Reply:
fixed